src/repository/exercisesRepository.ts

```diff
@@ -325,6 +325,426 @@ export const db: Schema = {
+      Tu: 'lavas',
+      'Ela/Ele/Você': 'lava',
+      Nós: 'lavamos',
+      'Eles/Elas/Vocēs': 'lavam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to end',
+      infinitive: 'acabar',
+      Eu: 'acabo',
+      Tu: 'acabas',
+      'Ela/Ele/Você': 'acaba',
+      Nós: 'acabamos',
+      'Eles/Elas/Vocēs': 'acabam',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to catch',
+      infinitive: 'apanhar',
+      Eu: 'apanho',
+      Tu: 'apanhas',
+      'Ela/Ele/Você': 'apanha',
+      Nós: 'apanhamos',
+      'Eles/Elas/Vocēs': 'apanham',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to wear',
+      infinitive: 'vestir',
+      Eu: 'visto',
+      Tu: 'vestes',
+      'Ela/Ele/Você': 'veste',
+      Nós: 'vestimos',
+      'Eles/Elas/Vocēs': 'vestem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to divide',
+      infinitive: 'dividir',
+      Eu: 'divido',
+      Tu: 'divides',
+      'Ela/Ele/Você': 'divide',
+      Nós: 'dividimos',
+      'Eles/Elas/Vocēs': 'dividem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to take off',
+      infinitive: 'despir',
+      Eu: 'dispo',
+      Tu: 'despes',
+      'Ela/Ele/Você': 'despe',
+      Nós: 'despimos',
+      'Eles/Elas/Vocēs': 'despem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to correct',
+      infinitive: 'despir',
+      Eu: 'corrijo',
+      Tu: 'corriges',
+      'Ela/Ele/Você': 'corrige',
+      Nós: 'corrigimos',
+      'Eles/Elas/Vocēs': 'corrigem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to decide',
+      infinitive: 'decidir',
+      Eu: 'decido',
+      Tu: 'decides',
+      'Ela/Ele/Você': 'decide',
+      Nós: 'decidimos',
+      'Eles/Elas/Vocēs': 'decidem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to get',
+      infinitive: 'conseguir',
+      Eu: 'consigo',
+      Tu: 'consegues',
+      'Ela/Ele/Você': 'consegue',
+      Nós: 'conseguimos',
+      'Eles/Elas/Vocēs': 'conseguem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to serve',
+      infinitive: 'sirvo',
+      Eu: 'consigo',
+      Tu: 'serves',
+      'Ela/Ele/Você': 'serve',
+      Nós: 'servimos',
+      'Eles/Elas/Vocēs': 'servem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to feel',
+      infinitive: 'sentir',
+      Eu: 'sinto',
+      Tu: 'sentes',
+      'Ela/Ele/Você': 'sente',
+      Nós: 'sentimos',
+      'Eles/Elas/Vocēs': 'sentem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to translate',
+      infinitive: 'traduzir',
+      Eu: 'traduzo',
+      Tu: 'traduzes',
+      'Ela/Ele/Você': 'traduz',
+      Nós: 'traduzimos',
+      'Eles/Elas/Vocēs': 'traduzem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to translate',
+      infinitive: 'traduzir',
+      Eu: 'traduzo',
+      Tu: 'traduzes',
+      'Ela/Ele/Você': 'traduz',
+      Nós: 'traduzimos',
+      'Eles/Elas/Vocēs': 'traduzem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to leave',
```

Comment:
yeah but this was more on poetry. 

if you want to use **partir** is more translated to "to break" - a plate, a leg, ...
if you want **to leave** then is more combining the verb **to go** + away :
eu vou-me embora
tu vais-te embora
ele vai-se embora
nos vamo-nos embora
eles vão-se embora

there will be more examples where you combine words to match an English verb

Reply:
removing